GObject-2.0.gir

```diff
@@ -2583,7 +2583,7 @@ after the object is is freed.</doc>
 the first parameter.
 
 @destroy_data will be called as a finalize notifier on the #GClosure.</doc>
-        <return-value transfer-ownership="none">
+        <return-value transfer-ownership="full">
```

Comment:
Eh...

Reply:
`git bisect` gives https://gitlab.gnome.org/GNOME/gobject-introspection/-/commit/918a7720a03a8284a9c1e2626a62d4f8930f633d but that doesn't seem to make much sense

Reply:
https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/377